run_tests.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
```

Comment:
Would want to remove the existing run_tests.sh and run this script instead in CI? And remove the stuff from CI that does installation, since this script would do that in the virtualenv?